presidio-image-redactor/presidio_image_redactor/dicom_image_pii_verify_engine.py

```diff
@@ -45,12 +45,14 @@ def verify_dicom_instance(
         self,
         instance: pydicom.dataset.FileDataset,
         padding_width: int = 25,
+        display_image: bool = True,
         **kwargs,
     ) -> Tuple[PIL.Image.Image, dict, list]:
         """Verify PII on a single DICOM instance.
 
         :param instance: Loaded DICOM instance including pixel data and metadata.
         :param padding_width: Padding width to use when running OCR.
+        :param display_image: If the verificationimage is displayed and returned.
```

Comment:
displayed? is it displayed in the ImagePiiVerifyEngine?

Reply:
If `self.verify` is called, then the image is displayed as well as returned.